client/client-multi/client-objective-c/KaaTests/KaaClientTest.m

```diff
@@ -23,6 +23,38 @@
 #import <XCTest/XCTest.h>
 #import <Kaa/Kaa.h>
 
+
+@interface ClientStateDelegateFailure : NSObject
```

Comment:
I'd rather put there a name that will give an idea of possible issue. (e.g. ClientStateDelegateWithoutImplementedMethods)